### PR TITLE
OCM-7432 | Defining WIF resources and endpoints

### PR DESCRIPTION
**Pull request by renan-campos**

### CODE REVIEW

File: model/clusters_mgmt/v1/wif_config_type.model [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	IssuerUrl String
+	Jwks string
+}
+
+struct WifTime {
+	// Date and time when the wif_config was initially created.
+	CreatedAt Date
+	// Date and time when the wif_config was last checked.
+	LastChecked Date
+}
```
Comment (renan-campos) [inline model/clusters_mgmt/v1/wif_config_type.model:64]: To be removed.

File: model/clusters_mgmt/v1/wif_config_type.model [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+	IdentityProviderId String
+	IssuerUrl String
+	Jwks string
+}
+
+struct WifTime {
+	// Date and time when the wif_config was initially created.
+	CreatedAt Date
+	// Date and time when the wif_config was last checked.
+	LastChecked Date
```
Comment (ckandag) [inline model/clusters_mgmt/v1/wif_config_type.model:63]: Time is mainly to determine if a WifConfig is stale \ hanging around too long and needs to be cleanedup.
Given that LastUsed makes more sense than lastChecked.

Comment (renan-campos) [reply model/clusters_mgmt/v1/wif_config_type.model]: As discussed during office hours, this structure and its associated field within the WifConfig resource has been removed.

File: model/clusters_mgmt/v1/wif_config_type.model (hunk 1 above)
Comment (renan-campos) [reply model/clusters_mgmt/v1/wif_config_type.model]: removed.*///////////////

File: model/clusters_mgmt/v1/wif_config_type.model [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	ProjectId String
+	// The list of service accounts and their associated roles that will need to be
+	// configured on the user's GCP project.
+	ServiceAccounts []WifServiceAccount
+	// The workload identity configuration data that will be used to create the
+	// workload identity pool on the user's account.
+	WorkloadIdentityPool WifWorkloadIdentityPool
+}
+
+struct WifWorkloadIdentityPool {
```
Comment (JakobGray) [inline model/clusters_mgmt/v1/wif_config_type.model:40]: After discussing shortening the table and column names for the database this Wif Workload Identity Pool name does seem redundant and could potentially just be WifPool

Comment (renan-campos) [reply model/clusters_mgmt/v1/wif_config_type.model:40]: updated the struct name.

**Review by vkareh — APPROVED** [review]
LGTM

**Pull request merged**